playground/src/components/entry-point/entry-point.spec.js

```diff
@@ -1,13 +1,56 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { MC_API_PROXY_TARGETS } from '@commercetools-frontend/constants';
+import { renderAppWithRedux } from '@commercetools-frontend/application-shell/test-utils';
 import { ApplicationStateMachines } from './entry-point';
 
-describe('rendering', () => {
-  let wrapper;
-  beforeEach(() => {
-    wrapper = shallow(<ApplicationStateMachines />);
+const createStateMachinesListSdkMock = () => ({
+  action: {
+    type: 'SDK',
+    payload: {
+      method: 'GET',
+      service: 'states',
+      options: {
+        perPage: 25,
+        page: 1,
+      },
+      mcApiProxyTarget: MC_API_PROXY_TARGETS.COMMERCETOOLS_PLATFORM,
+    },
+  },
+  response: {
+    count: 25,
+    offset: 0,
+    total: 2,
+    results: [],
+  },
+});
+
+const render = (options) => {
+  return renderAppWithRedux(<ApplicationStateMachines />, {
+    sdkMocks: [createStateMachinesListSdkMock()],
+    permissions: {
+      canViewDeveloperSettings: true,
+      canManageDeveloperSettings: true,
+    },
+    ...options,
+  });
+};
+
+describe('when route is /state-machines', () => {
+  let rendered;
+  it('should render state machines', async () => {
+    rendered = render({
+      route: '/project/state-machines',
+    });
+    await rendered.findByText(/State machines/i);
   });
-  it('should render state-machines route', () => {
-    expect(wrapper).toRender({ path: '/:projectKey/state-machines' });
```

Comment:
How dare you remove this meaningful test!

Reply:
my fears has come true (tomatoes on face)